src/controllers/AuthController.ts

```diff
@@ -0,0 +1,68 @@
+import { Request, Response } from 'express'
+import bcrypt from 'bcrypt'
+import { createUserQuery, getUserQuery } from '../queries/UserQuery'
+import authSchema from '../validation/userValidation'
+import { TokenGen } from '../helpers/AuthHelper'
+import { Validator } from '../validation/validator'
+
+// export const createUser = () => {createUserQuery}
+
+export const login = (req: Request, res: Response) => {
+  const tokenGen = new TokenGen({
```

Comment:
this should be coming from a query! insert a dummy user into your database and make this controller generic not depending on any dummy test cases